Userland/Libraries/LibPDF/Fonts/PDFFont.cpp

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "AK/StringView.h"
```

Comment:
`#include <AK/StringView.h>`

Reply:
VSCode is putting these always by default with "", it's easy to *not* see them. I'll see if I can toggle somehow this behaviour to avoid it in the future.